src/plugins/timelion/public/panels/timechart/_index.scss

```diff
@@ -0,0 +1,67 @@
+.timChart {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+
+  // Custom Jquery FLOT / schema selectors
+  // Cannot change at the moment
```

Comment:
Since we are completely removing Flot, can we check this comment and remove the CSS here that are no longer required?

Reply:
As timelion app will be removed soon we replaced Flot only for vis_type_timelion.